themes/theme-ios11/locale/it-IT.json

```diff
@@ -244,19 +244,19 @@
   },
   "navigation": {
     "login_register": "Accedi / Registrati",
-    "your_account": "Il tuo Account",
+    "your_account": "Il mio account",
     "home": "Pagina iniziale",
     "categories": "Categorie",
     "my_orders": "I miei ordini",
     "cart": "Carrello spesa",
     "shipping": "Modalità di spedizione",
-    "about": "A riguardo",
+    "about": "Su di noi",
```

Comment:
auch hier Durcheinander in allen Sprachen: manchmal "Über uns", manchmal "Impressum" bei "about"